.travis.yml

```diff
@@ -96,7 +98,8 @@ before_install:
     if [[ "$UNSUPPORTED" == true ]]; then
       pip install 3to2
       export PIP_NO_COMPILE=1
-      sed -E -i.bak '/^(coala|mypy-lang)/d' requirements.txt bear-requirements.txt
+      sed -i.bak '/^coala/d' requirements.txt
+      sed -i.bak '/^mypy-lang/d' bear-requirements.txt
```

Comment:
this line needs to be either

`sed -i.bak '/^mypy-lang/d' requirements.txt`

or (if the above fails) ..

`sed -i.bak '/^mypy-lang/d' requirements.txt bear-requirements.txt`